site/content/extend/plugins/integrations_directory_process.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Listing on integrations.mattermost.com
+date: 2018-10-01T00:00:00-05:00
+subsection: Plugins (Beta)
+weight: 50
+---
+
+
+This document outlines the process of getting your plugin onto the [Integrations Directory](https://integrations.mattermost.com).  
+
+When submitting an Integration, keep the following in mind:
+- An Integration Directory Listing can be as simple as a blog post with instructions on how to configure an integration manually, a script, a plugin for another platform that connects with Mattermost, or a native Mattermost plugin.  
+- Mattermost staff will review your plugin for functionality and clarity. If the description of your plugin is not clear, we will help by submitting a Pull Request with some suggestions.
+- If possible, please enable the "Issues" tab in Github so users can contribute and collaborate on the integration.
+- Please be responsive to our questions or suggestions, we curate the integrations directory for Mattermost admins to easily find useful integrations and will work with you to make sure your listing is accurate.
+- Mattermost may not have the software available to test an integration, in which case we could do a review over a screenshare to verify functionality.
+- Your submission doesn't guarantee it will be accepted to the integrations directory.
```

Comment:
I'm curious why you added this. It seams discouraging to me. If my integration fulfils the checklist, why should it not get accepted?